backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/order/products_proposal/model/ProductsProposalRow.java

```diff
@@ -100,8 +102,13 @@ public static ProductsProposalRow cast(final IViewRow row)
 	@ViewColumn(seqNo = 70, fieldName = FIELD_IsCampaignPrice, captionKey = "IsCampaignPrice", widgetType = DocumentFieldWidgetType.YesNo)
 	private final boolean isCampaignPrice;
 
+	/**
```

Comment:
didn't this work with `ViewColumnHelper` and `Displayed.SYSCONFIG`?

Reply:
nope, as the ProductsProposalRow is shared between multiple modals so the fields are programmatically added via `de.metas.ui.web.view.descriptor.ViewLayout.Builder#addElementsFromViewRowClassAndFieldNames`.. a solution would be to refactor everything such that every modal gets its own row definition.